AC-Meeting-Guidelines.md

```diff
@@ -0,0 +1,38 @@
+# Architecture Committee Meeting Guidelines
+
+The Architecture Committee (AC) recommends following a set of guidelines for raising issues and adding topics to the meeting agenda.
+
+## Use a GitHub issue or an email to the mailing list first
+
+For a project like Kata Containers, with contributors spanning across so many different time zones, asynchronous discussions and reviews are the preferred communication media.
+
+Directly starting a proposal or trying to resolve new technical issues through the AC meeting has many drawbacks:
+1. It potentially leaves a large part of the community out of the initial dicussion.
+1. It does not give the AC meeting audience enough time to think through the proposal and give constructive and well thought feedback.
+1. It makes it hard to track and share the discussion outcome.
+
+As a consequence, the AC recommends using GitHub issues and PRs, or the kata-dev mailing list at first before adding new items to the AC meetings agenda, especially if you think the topic could be discussed and resolved in less than two weeks.
+
+## AC Agenda topics must be supported by a GitHub issue or PR, or an email sent to the mailing list
+
+Except for a few cases listed in the next section, any topic brought to the AC meeting must be linked to a GitHub issue, PR, or to an email to the mailing list.
```

Comment:
For convenience, maybe add a link to the mailing list (http://lists.katacontainers.io/cgi-bin/mailman/listinfo/kata-dev) and maybe even to the issue creation link (https://github.com/kata-containers/kata-containers/issues)?

Reply:
Do we want a new template in issues for the topics you listed below? Something like "Community discussion" with a template that suggests:

* Are you asking for a solution, a resolution, a go/no-go or feedback?
* Is there a deadline to your request?

Reply:
> For convenience, maybe add a link to the mailing list (http://lists.katacontainers.io/cgi-bin/mailman/listinfo/kata-dev) and maybe even to the issue creation link (https://github.com/kata-containers/kata-containers/issues)?

Both links added.